website/docs/reference/resource-configs/postgres-configs.md

```diff
@@ -108,21 +108,43 @@ models:
 
 ## Materialized views
 
-The Postgres adapter supports [materialized views](https://www.postgresql.org/docs/current/rules-materializedviews.html).
-Indexes are the only configuration that is specific to `dbt-postgres`.
-The remaining configuration follows the general [materialized view](/docs/build/materializations#materialized-view) configuration.
-There are also some limitations that we hope to address in the next version.
+The Postgres adapter supports [materialized views](https://www.postgresql.org/docs/current/rules-materializedviews.html)
+with the following configuration parameters:
 
-### Monitored configuration changes
+| Parameter                 | Type       | Required | Default   | Change Monitoring Support | Reference           |
+|---------------------------|------------|----------|-----------|---------------------------|---------------------|
+| `on_configuration_change` | STRING     | NO       | `'apply'` | N/A                       |                     |
+| `indexes`                 | LIST[DICT] | NO       |           | ALTER                     | [Indexes](#indexes) |
```

Comment:
The table can be one column thinner if doing a hyperlink like [`indexes`](#indexes) in the first column instead.

Reply:
Fair. Let me play with it and see how it looks. When I started the docs updates I didn't have a link for everything, so it felt like an after thought. but at this point, I think we do have one, so it's probably better to have it up front.